### ISSUE TITLE
Invalid number of query parameters

### ISSUE THREAD

**Issue by ngocdangrby** [wso2-extensions/siddhi-execution-tensorflow#21]
Hi Siddhi Team,
I had meet some problems when trying to execute this guide: ["Realtime predictions with pre-trained ML models"](https://siddhi.io/en/v5.1/docs/guides/realtime-movie-recommendation/guide/)
**Description:**
***Prob 1:***
Two links may  attached wrong files (in Mandatory Requirements part):
- Pre-trained sentiment analysis tensorflow model
- Pre-trained movie recommendation PMML model

***Prob 2:***
Siddhi show this error
`Line: 21. Position: 24 and @ Line: 22. Position: 57. Invalid number of query parameters. Number of inputs and number of outputs are specified as 4 and 2 respectively. So the total number of query parameters should be 11 but 10 given.`
When i used this query:
`From FeedbackInputStream#tensorFlow:predict('/home/dang/siddhi/text_input_model/1569807768','input_ids','input_mask', 'label_ids','segment_ids',
'labels', 'probabilities',userId, movieId, reviewComment) `
I think that if has 4 inputs and 2 outputs, so total number of parameters should be 4(input)+2(output)+ 1(model_path) = 7 + number of other attributes
Moreover, *input.node.names* and *output.node.names* is optional (follow[ this doc](https://siddhi-io.github.io/siddhi-execution-tensorflow/api/latest/)), so number of parameters must not exactly 11.
Below is SignatureDef of your .pb model

![SignatureDef](https://i.imgur.com/hKA4VVs.png)

Hope your reply soon.

**Comment by pcnfernando** [comment]
What is the siddhi-execution-tensorflow version you are trying out?

**Comment by ngocdangrby** [comment]
Thank for your quick response.
I am using  siddhi-execution-tensorflow 2.0.2
I tried use version 2.0.1 but backend app crash when app trying to load bert model in this guide
